src/solvers/dgsem_unstructured/dg_2d.jl

```diff
@@ -417,12 +418,12 @@ end
 # Note, it is necessary to set and add in the nonconservative values because
 # the upper left/lower right diagonal terms have been peeled off due to the use of
 # `derivative_split` from `dg.basis` in [`flux_differencing_kernel!`](@ref)
-@inline function calc_boundary_flux!(surface_flux_values, t, boundary_condition,
+@inline function calc_boundary_flux!(surface_flux_values, t, boundary_condition::BC,
                                      mesh::UnstructuredMesh2D,
                                      nonconservative_terms::True, equations,
                                      surface_integral, dg::DG, cache,
                                      node_index, side_index, element_index,
-                                     boundary_index)
+                                     boundary_index) where {BC}
```

Comment:
Same here - `boundary_condition` is called below.

Reply:
See above